docs/reference_glossary.md

```diff
@@ -78,6 +78,11 @@ A feature that allows you to tap the leader key followed by a sequence of 1, 2,
 
 * [Leader Key Documentation](feature_leader_key.md)
 
+## Compose Onboard
+Allows composing directly on the keyboard. After pressing the compose onboard key, the following keystrokes trigger a preconfigured action like pressing other keys, inserting a character, or any other quantum features.
+
+* [Compose Onboard Documentation](feature_compose_onboard.md)
+
```

Comment:
I don't know that this needs to be added to the glossary, but if its decided that it should be so, this section should move up between Compile and Dvorak, as this page is organized alphabetically.